packages/e2e-tests/specs/editor/various/block-editor-keyboard-shortcuts.test.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	createNewPost,
+	clickBlockAppender,
+	getEditedPostContent,
+	pressKeyWithModifier,
+} from '@wordpress/e2e-test-utils';
+
+const createTestParagraphBlocks = async () => {
+	await clickBlockAppender();
+	await page.keyboard.type( 'First paragraph' );
+	await page.keyboard.press( 'Enter' );
+	await page.keyboard.type( 'Second paragraph' );
+	await page.keyboard.press( 'Enter' );
+	await page.keyboard.type( 'Third paragraph' );
+};
+
+describe( 'block editor keyboard shortcuts', () => {
+	beforeEach( async () => {
+		await createNewPost();
+	} );
+
+	describe( 'move blocks', () => {
+		const moveUp = async () => pressKeyWithModifier( 'secondary', 't' );
+		const moveDown = async () => pressKeyWithModifier( 'secondary', 'y' );
+		const multiSelectBlocks = async () => {
+			await page.keyboard.down( 'Shift' );
+			await page.keyboard.press( 'ArrowUp' );
+			await page.keyboard.up( 'Shift' );
+		};
```

Comment:
`moveUp` and `moveDown` are unequivocal in their effects, and there is only one possible means to trigger each of them, so they can make sense as functions.

In contrast, `multiSelectBlocks` isn't unique: there are many ways to cause multiple blocks to be selected, and so this function depends on specific motions (in this case, upward). This means that when a particular test calls `multiSelectBlocks`, it's not clear for the reader what is happening in terms of editor state, unless they go up to read the definition of `multiSelectBlocks`.

This isn't a hard "no", but it's my own interpretation of what can improve the readability of tests.

Reply:
You're not wrong about this. I changed it be more readable.